examples/gcal_oo_or.ty

```diff
@@ -190,28 +201,26 @@ topo.sim.connect(
 topo.sim.connect(
     'LGNOn','V1',delay=0.05,strength=p.aff_strength,name='LGNOnAfferent',
     dest_port=('Activity','JointNormalize','Afferent'),
-    connection_type=projection.CFProjection,learning_rate=p.aff_lr,
+    connection_type=GPUSparseCFProjection if p.gpu else projection.CFProjection,learning_rate=p.aff_lr,
     nominal_bounds_template=sheet.BoundingBox(radius=0.27083),
-    weights_generator=pattern.random.GaussianCloud(gaussian_size=2*0.27083),
-    learning_fn=learningfn.optimized.CFPLF_Hebbian_opt())
+    weights_generator=pattern.random.GaussianCloud(gaussian_size=2*0.27083))
 
 topo.sim.connect(
     'LGNOff','V1',delay=0.05,strength=p.aff_strength,name='LGNOffAfferent',
     dest_port=('Activity','JointNormalize','Afferent'),
-    connection_type=projection.CFProjection,learning_rate=p.aff_lr,
+    connection_type=GPUSparseCFProjection if p.gpu else projection.CFProjection,learning_rate=p.aff_lr,
     nominal_bounds_template=sheet.BoundingBox(radius=0.27083),
-    weights_generator=pattern.random.GaussianCloud(gaussian_size=2*0.27083),
-    learning_fn=learningfn.optimized.CFPLF_Hebbian_opt())
+    weights_generator=pattern.random.GaussianCloud(gaussian_size=2*0.27083))
 
 topo.sim.connect(
     'V1','V1',delay=0.05,strength=p.exc_strength,name='LateralExcitatory',
-    connection_type=projection.CFProjection,learning_rate=p.exc_lr,
+    connection_type=GPUSparseCFProjection if p.gpu else projection.CFProjection,learning_rate=p.exc_lr,
     nominal_bounds_template=sheet.BoundingBox(radius=0.104),
     weights_generator=pattern.Gaussian(aspect_ratio=1.0, size=0.05))
 
 topo.sim.connect(
     'V1','V1',delay=0.05,strength=-1.0*p.inh_strength,name='LateralInhibitory',
-    connection_type=projection.CFProjection,learning_rate=p.inh_lr,
```

Comment:
Please make a variable for ``projection_type`` just as you did for ``sheet_type`` above, to remove all these ``if`` expressions.